docs/source/reference/helm-values-spec.rst

```diff
@@ -0,0 +1,832 @@
+.. _helm-values-spec:
+
+SkyPilot API Server Helm Chart Values
+=====================================
+
+The SkyPilot API server helm chart provides typical `helm values <https://helm.sh/docs/chart_template_guide/values_files/>`_ as configuration entries. Configuration values can be passed in two ways when installing the chart:
+
+* ``--values`` (or ``-f``): Specify a YAML file with overrides. This can be specified multiple times and the rightmost file will take precedence;
+
+  .. code-block:: bash
+
+      cat <<EOF > values.yaml
+      apiService:
+        image: berkeleyskypilot/skypilot:0.9.1
+      EOF
+
+      helm install $RELEASE_NAME skypilot/skypilot-nightly --devel --values values.yaml
+
+* ``--set``: Specify overrides on the command line.
+
+  .. code-block:: bash
+      
+      helm install $RELEASE_NAME skypilot/skypilot-nightly --set apiService.image="berkeleyskypilot/skypilot:0.9.1"
```

Comment:
For future: We may want to rename the helm chart name to be `skypilot/skypilot-api-server` to make it more explicit, as `skypilot-nightly` is a bit confusing with our pypi package.

Reply:
Followup: https://github.com/skypilot-org/skypilot/issues/5431